Common/src/main/java/dk/gruppeseks/bodtrd/common/data/entityelements/Path.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package dk.gruppeseks.bodtrd.common.data.entityelements;
+
+/**
+ *
+ * @author Frederik
+ */
+public class Path
+{
+    private Position[] _path;
+    private int nextPosition;
+
+    public Path(Position[] path)
+    {
+        _path = path;
+    }
+
+    public Position[] _getTotalPath()
+    {
+        return _path;
+    }
+
+    public Position peekNextPosition()
```

Comment:
this for instance should be named peek or peekPosition and the internal index should just be position imo